eventActions/highlightActions.js

```diff
@@ -18,7 +18,7 @@ class highlightActions {
 				let currentPhrase = result[i].phrase;
 				let currentId = result[i].users;
 				// Check if the message.contains(that phrase)
-				if(message.content.toLowerCase().includes(currentPhrase)){
+				if(message.content.toLowerCase().includes(' ' + currentPhrase + ' ')){
```

Comment:
Will this work if the first word in a message is the tracked word?

Reply:
No- nor at the end of a message or with punctuation either.
I asked in R&D but would those use cases be worth adding? Begin/end is easy, punctuation a bit more annoying